packages/jni-swig-stub/realm.i

```diff
@@ -222,18 +222,22 @@ bool throw_as_java_exception(JNIEnv *jenv) {
 // Type map for int64_t has an erroneous cast, don't know how to fix it except with this
 %typemap(in) void** ( jlong *jarr ){
     // Original
-    %#if defined(__ANDROID__)
+    %#if defined(__ANDROID__) && defined(__aarch64__) // Android arm64-v8a
```

Comment:
This is also x86_64, right?

Reply:
no `armeabi-v7a, x86_64 and x86` is handled in the next branch